extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/PersistenceXmlDescriptorBuildItem.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.hibernate.orm.deployment;
+
+import org.hibernate.jpa.boot.internal.ParsedPersistenceXmlDescriptor;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Provides instances of {@see ParsedPersistenceXmlDescriptor}, the raw representation
+ * of a persistence.xml file as it is after being located and parsed.
+ * Exposed as a possible integration API: other extensions can produce additional
+ * configuration instances.
+ *
+ * @author Sanne Grinovero <sanne@hibernate.org>
```

Comment:
It would be nice to drop the author tags since we haven't been using them in a while :)